guake/main.py

```diff
@@ -42,7 +42,11 @@
 log = logging.getLogger(__name__)
 
 # Force use X11 backend under wayland before any import of GDK through dependencies
+# This could fix weird problems under Wayland
+# But if user set GUAKE_ENABLE_WAYLAND=1, then force use Wayland backend
 os.environ["GDK_BACKEND"] = "x11"
+if os.environ["GUAKE_ENABLE_WAYLAND"] == 1:
```

Comment:
We don't need that env variable exactly be `1`, it can be `if set then use wayland`:

```
if "GUAKE_ENABLE_WAYLAND" in os.environm:
  ....
```

Reply:
OK, edited.